pxtlib/service.ts

```diff
@@ -158,6 +158,7 @@ namespace ts.pxtc {
         advanced?: boolean;
         deprecated?: boolean;
         useEnumVal?: boolean; // for conversion from typescript to blocks with enumVal
+        callInDebugger?: boolean; // for getters, they will be invoked by the debugger.
```

Comment:
callFromDebugger maybe?

Reply:
Already merged it in https://github.com/Microsoft/pxt-common-packages/pull/762, we can change it later if we feel it's not clear enough.